libs/template/renderer.go

```diff
@@ -255,7 +248,7 @@ func (r *renderer) walk() error {
 			if err != nil {
 				return err
 			}
-			logger.Infof(r.ctx, "added file to list of in memory files: %s", f.relPath)
+			logger.Infof(r.ctx, "added file to list of in memory files: %s", f.Path())
```

Comment:
Message needs an update now that not everything is in-memory.

Reply:
done